python/doc/examples/calibration/bayesian_calibration/plot_rwmh_python_distribution.py

```diff
@@ -192,3 +196,22 @@ def getParameter(self):
     graph.setLegends(['Posterior', 'Prior'])
     grid.setGraph(0, parameter_index, graph)
 _ = View(grid)
+
+# %%
+# Define an improper prior
+# --------------------------
+
+# %%
+# Now, define an improper prior which log pdf is defined by f(beta, alpha) = -log(beta)
+
+logpdf = ot.SymbolicFunction(['beta', 'alpha'], ['-log(beta)'])
+support = ot.Interval([0] * 2, [1] * 2)
+support.setFiniteUpperBound([False] * 2)
+
+# %%
+# Sample from the posterior distribution
+
+sampler2 = ot.RandomWalkMetropolisHastings(logpdf, support, initialState, instrumental)
+sampler2.setLikelihood(conditional, x)
+sample2 = sampler2.getSample(1000)
+print("Acceptance rate: %s" % (sampler2.getAcceptanceRate()))
```

Comment:
A graph like the one above would be helpful. Obviously the improper prior cannot be meaningfully drawn, but you could draw the posterior PDF.

Reply:
of course